validate_test.go

```diff
@@ -105,6 +105,36 @@ addr:
 				} `yaml:"addr" validate:"required"`
 			}{},
 		},
+		{
+			TestName: "Test Validation with wrong field type",
+			YAMLContent: `---
+name: myDocument
+roles:
+  name: myRole
+  permissions:
+	- hello
+	- how
+	- are
+	- you
+	`,
+			ExpectedErr: `[4:7] Mapping node found where Sequence is expected
```

Comment:
I want to use the YAML structure name instead of the node name.
For example, in this case I would like to print like `mapping was used where sequence is expected` message .

Reply:
I made the change to use the YAML structure name.  However now the CI build is failing.

Reply:
The failure message makes it seem like the commit is missing an author:
> fatal: empty ident name (for <circleci@c2212a33bbf2>) not allowed

Reply:
Never mind it seems to have been a merge conflict 😄